src/github.com/getlantern/fronted/direct.go

```diff
@@ -117,6 +117,24 @@ func (d *direct) NewDirectHttpClient() *http.Client {
 	}
 }
 
+// Do continually retries a given request until it succeeds because some fronting providers
+// will return a 403 for some domains.
+func (d *direct) Do(req *http.Request) (*http.Response, error) {
+	for i := 0; i < 6; i++ {
+		client := d.NewDirectHttpClient()
+		if resp, err := client.Do(req); err != nil {
+			log.Errorf("Could not complete request %v", err)
+			continue
+		} else if resp.StatusCode > 199 && resp.StatusCode < 400 {
+			return resp, err
+		} else {
+			_ = resp.Body.Close()
+			continue
+		}
```

Comment:
We should be able to get rid of these ``continue`` statements and we can just change ``_ = resp.Body.Close()`` to ``res.Body.Close()`` since we don't care about the return value (or is that to just be explicit that we're ignoring it?)